packages/opentelemetry-node/docs/metrics.asciidoc

```diff
@@ -1,30 +1,49 @@
----
-id: otelNodeMetrics
-slug: /otel-nodejs/metrics
-title: Metrics
----
+////
+Goal of this doc:
+???
 
-## Enabled by default
+Assumptions we're comfortable making about the reader:
+- ???
+- ???
+- ???
+////
```

Comment:
Can we run with this metrics doc as is for now? I know it is light on details.
We have plans to change what metrics are collected by default before GA.